src/d3a/models/strategy/external_strategies/__init__.py

```diff
@@ -88,6 +88,28 @@ def _unregister(self, payload):
     def register_on_market_cycle(self):
         self.connected = self._connected
 
+    def _device_stats(self, _):
```

Comment:
I think we have to be careful now with the naming.
On several point in the code we have device_stats, device_statistics and similar. 
IMO at this point here (and for the endpoint channel) a name like device_info would fit more. But i am of course open for other names two. What do you think?

Reply:
I agree, and you are right, this is not device_statistics that we are sending here. It is actually the current device information (essentially information about the current available/required energy). Changing this endpoint to device_info in d3a-web too. Thanks